delivery_postlogistics/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.update_module_names(
+        env.cr,
+        [("delivery_carrier_label_postlogistics", "delivery_postlogistics")],
+        merge_modules=True,
+    )
+
+    # Model renaming
+    # delivery.carrier.template.otion => postlogistics.delivery.carrier.template.option
+    rename_model_list = [
+        (
+            "delivery.carrier.template.option",
```

Comment:
I'm not sure this is correct. This model was defined `base_delivery_carrier_label` and you could have many *_carrier_label modules installed.
Probably we should simply drop the model and rely on the fact that all other carrier modules will register their own records.
@yvaucher any advice on this?